core/src/main/java/io/questdb/griffin/SqlCompilerImpl.java

```diff
@@ -404,10 +404,7 @@ private int addColumnWithType(AlterOperationBuilder addColumn, CharSequence colu
         CharSequence tok;
         tok = expectToken(lexer, "column type");
 
-        int type = ColumnType.tagOf(tok);
-        if (type == -1) {
-            throw SqlException.$(lexer.lastTokenPosition(), "invalid type");
-        }
+        int type = SqlUtil.toPersistedType(tok, lexer.lastTokenPosition());
```

Comment:
Nit: this is column type tag
```suggestion
        short type = SqlUtil.toPersistedColumnTypeTag(tok, lexer.lastTokenPosition());
```